sdk/eventhub/Azure.Messaging.EventHubs/stress/Directory.Build.props

```diff
@@ -0,0 +1,14 @@
+﻿<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Force project references to Azure clients.  This is necessary because the stress project
+    uses a project reference to the Event Hubs core library and the processor library.  The
+    processor library takes a package reference on the Event Hubs core library, leading versioning
+    to get confused in some scenarios.
+  -->
+  <PropertyGroup Condition="'$(UseProjectReferenceToAzureClients)' == ''">
+    <UseProjectReferenceToAzureClients>true</UseProjectReferenceToAzureClients>
```

Comment:
If you want this to be forced, do you need the condition, or is this intended to only force if not configured?

Reply:
The latter.   I don't want to block someone explicitly mixing in packages and opting out.   This is just meant to force the transitive dependencies to play nice in the default scenario.